documentation/docs/pipelines/abapEnvironment/configuration.md

```diff
@@ -0,0 +1,132 @@
+# Configuration
+
+In this section, you can learn how to create a configuration in a (GitHub) repository to run an ABAP Environment Pipeline.
+
+## 1. Prerequisites
+
+* Configure your Jenkins Server according to the [documentation](https://sap.github.io/jenkins-library/guidedtour/).
+* Create a git repository on a host reachable by the Jenkinsserver (e.g. GitHub.com). The pipeline will be configured in this repository.
+
+## 2. Jenkinsfile
+
+Create a file named `Jenkinsfile` in your repository with the following content:
+
+```
+@Library('piper-lib-os') _
+
+abapEnvironmentPipeline script: this
+```
+
+The annotation `@Library('piper-lib-os')` is a reference to the Jenkins Configuration, where you configured the Piper Library as a "Global Pipeline Library". If you want to **avoid breaking changes** we advise you to use a specific release of the Piper Library instead of the default master branch (see [documentation](https://sap.github.io/jenkins-library/customjenkins/#shared-library)).
+
+## 3. Manifest for Service Creation
+
+Create a file `manifest.yml`. The pipeline will create a SAP Cloud Platform ABAP Environment System in the beginning (and delete it in the end). This file describes the ABAP instance, which will be created:
+
+```yaml
+---
+create-services:
+- name:   "abapEnvironmentPipeline"
+  broker: "abap"
+  plan:   "16_abap_64_db"
+  parameters: "{ \"admin_email\" : \"user@example.com\", \"description\" : \"System for ABAP Pipeline\" }"
+```
+
+The example values are a suggestion. Please change them accordingly and don't forget to enter your own email address. Please be aware that creating a SAP Cloud ABAP Environment instance may incur costs.
+
+Please have a look at the [step documentation](https://sap.github.io/jenkins-library/steps/cloudFoundryCreateService/) for more details.
```

Comment:
I would suggest to point out that this file & step is optional, if you already have a system configured. Or point out in the `Configuration` section which steps can be left out if you already have a system available with the appropriate user & communication scenario,

What do you think?

Reply:
Yes, this is a good suggestions. However this is a different scenario, which could be documented in the future. At the moment, I want to focus on the scenario including cloud foundry.

Reply:
That was also my point of concern. As you said, I think this would rather be a thing for future documentation